components/eam/src/chemistry/yaml/aero_model/f90_yaml/aero_model_wetdep_end_yml.f90

```diff
@@ -0,0 +1,70 @@
+#ifdef YAML_AERO_MODEL_WETDEP
+  if(yaml%flag_print) then ! if this column exists in lchnk
+
+     !write output header
+     call write_output_header(unit_output)
+
+     !start writing data
+
+        ! write out fields that are set within intent-out structure 'cam_out'
+
```

Comment:
let's also write `aerdepwetcw` and `aerdepwetis`.

Reply:
These fields have also been added.